### DISCUSSION TITLE
Why would CQ Zone be recorded incorrectly?

### DISCUSSION THREAD

**Discussion by ei8fdb** [magicbug/Cloudlog#3211] [Q&A] [answered]
I'm curious how does Cloudlog work out a DX callsign's CQ Zone.

8 of my QSOs yesterday were recorded with the incorrect CQ Zone. I've seen this in the past and I've fixed them manually by editing the QSO details.

Cloudlog knows the callsign (in this case all were operating in their home countries, so no possibly confusing / cases), I'm confused why would CQ Zone be logged incorrectly? 

I assumed it would "look it up locally" (:smiley:) , but now I'm thinking it's more complicated than that!

Is there anything I can do when logging to prevent this? Thanks for Cloudlog.

**Comment by magicbug** [comment]
Its roughly based on clublog data and callbook returned information.. there's no exact art to it its best effort

**Comment by ei8fdb** [comment, marked as answer]
@magicbug  aha, that explains it. Thanks.
